src/target/source/interface_c.cc

```diff
@@ -152,6 +161,43 @@ class InterfaceCNode : public runtime::ModuleNode {
     code_stream << "#define " << macro_name_prefixed << " " << macro_value << "\n";
   }
 
+  void EmitConstantPool(std::stringstream& code_, const std::string& brief_description,
```

Comment:
We need a test for this in interface_c_test.cc

Reply:
added